### PR TITLE
LF-190: improve button clickability

### PR DESCRIPTION
**Pull request by Addminus**
Users have issues understanding what they should do when an action is required. This PR highlights the button they should click to resume a swap.

### CODE REVIEW

File: src/components/TransactionsTable.tsx [hunk 1]
```diff
@@ -60,20 +60,24 @@ function TrasactionsTable({
...
+          style={{ marginRight: 10, padding: '3px 16px 4px 16px' }}
+          type="primary"
           shape="round"
           onClick={() => selectRoute(route)}>
           Resume Swap
         </Button>
         <Popconfirm
           title={<>Are you sure to delete this transfer?</>}
           onConfirm={() => deleteRoute(route)}
-          // onCancel={cancel}
```
Comment (mathiasmoeller) [inline src/components/TransactionsTable.tsx:72]: Can this be removed?

Comment (mathiasmoeller) [reply src/components/TransactionsTable.tsx:72]: what about the `cancel` function? Can it be removed as well?

File: src/components/TransactionsTable.tsx [hunk 2]
```diff
@@ -60,20 +60,24 @@ function TrasactionsTable({
...
           okText="Yes"
           okType="danger"
           cancelText="No">
-          <Button danger type="ghost" shape="round" onClick={() => {}}>
+          <Button
+            style={{ padding: '3px 16px 4px 16px' }}
+            danger
+            type="ghost"
+            shape="round"
+            onClick={() => {}}>
```
Comment (mathiasmoeller) [inline src/components/TransactionsTable.tsx:80]: What is this dummy button for?

File: src/components/TransactionsTable.tsx [hunk 3]
```diff
@@ -122,11 +126,6 @@ function TrasactionsTable({
       firstStep.execution?.process?.[0]?.startedAt &&
       new Date(firstStep.execution.process[0].startedAt).toLocaleString()
     const lastStep = route.steps[route.steps.length - 1]
-    // const firstAction = firstStep.action
-    // const firstEstimate = firstStep.estimate
-    // const lastAction = lastStep.action
-    // const lastEstimate = lastStep.estimate
```
Comment (mathiasmoeller) [inline src/components/TransactionsTable.tsx:128]: Can this be removed?

File: src/components/TransactionsTable.tsx (hunk 1 above)
Comment (Addminus) [reply src/components/TransactionsTable.tsx:72]: yep

**Review by mathiasmoeller — APPROVED** [review]

**Pull request merged**